gulpfile.js

```diff
@@ -25,8 +25,8 @@ const bower = gulp.task('bower', async function () {
     'license',
     'keywords'
   ];
-
-  for (let i = 0, l = fields.length; i < l; i++) {
+ var i=0,l;
+  for (i ,l = fields.length; i < l; i++) {
```

Comment:
Why use 'var'? This will interfere with other variables declared in a 'for' loop with `i` and `l`, as 'var' has a global scope.